### ISSUE TITLE
Jasper 0.9.1 has nothing on the screen

### ISSUE THREAD

**Issue by jiska** [jasperapp/jasper#139]
Nice to meet you.

When I run Jasper 0.9.1, nothing appears on the screen and it's blank.
It works fine in Jasper 0.9.0.

## The environment I'm using.

```bash
ProductName: Mac OS X
ProductVersion: 10.15.5
BuildVersion: 19F101
```

**Comment by h13i32maru** [comment]
I can't reproduce the problem 🤔 
Would you like to write a startup log?
```
/Applications/Jasper.app/Contents/MacOS/Jasper
```

For the time being, please use the old Jasper.
https://github.com/jasperapp/jasper/releases/tag/v0.9.0

**Comment by tunepolo** [comment]
In my environment, following messages are displayed.

```
❯ /Applications/Jasper.app/Contents/MacOS/Jasper
[N] [2020-07-08T15:55:38.23Z] [index.js:45:24] user data path: /Users/r0311/Library/Containers/io.jasperapp/data/Library/Application Support/jasper
[N] [2020-07-08T15:55:38.30Z] [index.js:46:24] app data path: /Users/r0311/Library/Application Support
[N] [2020-07-08T15:55:38.30Z] [index.js:47:24] config path: /Users/r0311/Library/Containers/io.jasperapp/data/Library/Application Support/jasper/io.jasperapp/config.json
[E] [2020-07-08T15:55:38.140Z] [index.js:50:28] Unhandled Rejection at: [object Promise]
[E] [2020-07-08T15:55:38.140Z] [index.js:51:28] reason: TypeError: Error processing argument at index 1, conversion failure from undefined
```

**Comment by h13i32maru** [comment]
I was able to reproduce it.
The cause was the restoration of the window state. I will fix it in the next version.
https://github.com/jasperapp/jasper/compare/v0.9.0...v0.9.1#diff-f41e9d04a45c83f3b6f6e630f10117feR102

**Comment by h13i32maru** [comment]
I released v0.9.2 which is fixed this crash bug. Please update.
https://jasperapp.io/release.html

**Comment by tunepolo** [comment]
New version works in my environment, thanks :-)
